lib/routes/account.js

```diff
@@ -1364,9 +1475,13 @@ module.exports = function (
                 )
                 .then(
                   function () {
-                    log.event('delete', request, {
+                    return log.event('delete', request, {
                       uid: emailRecord.uid.toString('hex') + '@' + config.domain
                     })
```

Comment:
Not for this PR, but something to think about for a future refactoring...

We're doing a lot of passing objects around here, passing in a `metricsContext` to each route and them passing the `request` to a bunch of logging calls.  I wonder if we could clean this up by changing around our abstractions.  What if we have some Hapi plugin (or whatever it's called) that added a metrics object to the incoming request, so we could do something like:

```
return request.app.metrics.event('delete', { ...data... });
return request.app.metrics.activityEvent('account.signed', { ...data... });
```

Do you think something like that might come out cleaner overall?